packages/wallets/src/components/Base/WalletPasswordField/PasswordMeter.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
 export interface PasswordMeterProps {
-    strength?: number;
+    score?: number;
```

Comment:
@aizad-deriv if you have `export type Score` then why you use `number` here?

Reply:
Thanks for pointing it out will make the changes accordingly 👍🏼